fmriprep/viz/reports.py

```diff
@@ -154,20 +154,46 @@ def index_error_dir(self, error_dir):
                         else:
                             scope.set_extra(k, v)
                     scope.level = 'fatal'
-                    message = '%s: %s\n\n' % (node_name, gist)
-                    # 8192 is the message size limit - the important info is probably at the end
-                    message += exception_text[-(8192-len(message)):]
 
-                    # remove file paths
-                    fingerprint = re.sub(r"(/[^/ ]*)+/?", '', message)
-                    # remove words containing numbers
-                    fingerprint = re.sub(r"([a-zA-Z]*[0-9]+[a-zA-Z]*)+", '', fingerprint)
-                    # adding the return code if it exists
-                    for line in message.split('\n'):
-                        if line.startswith("Return code"):
-                            fingerprint += line
+                    # Group common events with pre specified fingerprints
+                    fingerprint_dict = {'permission-denied': ["PermissionError: [Errno 13] "
+                                                              "Permission denied"],
+                                        'memory-error': ["MemoryError", "Cannot allocate memory"],
+                                        'reconall-already-running': ["ERROR: it appears that "
+                                                                     "recon-all is already "
+                                                                     "running"],
+                                        'no-disk-space': ["OSError: [Errno 28] No space left on "
+                                                          "device", "[Errno 122] Disk quota "
+                                                                    "exceeded"],
```

Comment:
indentation

Reply:
what's wrong with it? (mind that `"exceeeded" is part of the list item from the previous line)

Reply:
I'd swear that it should be aligned right after the opening bracket, with `"device"`. That is taking into account that "exceeded" is part of the previous line. However, this would be more readable and use the same number of lines with:

```Python
                                        'no-disk-space': [
                                            "OSError: [Errno 28] No space left on device",
                                            "[Errno 122] Disk quota exceeded"],
```